### ISSUE TITLE
log4net 2.0.10/11 support

### ISSUE THREAD

**Issue by christo-djambov-axway** [net-commons/common-logging#185]
Hello there 
Is there planned support for log4net 2.0.10 and/or 2.0.11?
thank you

**Comment by DoomerDGR8** [comment]
log4net 2.0.12 is out. This needs to be updated.

**Comment by tompazourek** [comment]
I prepared a PR: #187

However, I'm not sure how to run the CI, publish the NuGet package, etc. I'm not one of the maintainers.

**Comment by CrazedBear** [comment]
Hi all,  @tompazourek  any news on this one?

**Comment by tompazourek** [comment]
@CrazedBear No, I'm not a maintainer of this repo, and nobody has looked at my PR contribution yet. There's not much more I can do.

For the project where I needed it, I ended up just copying the relevant code and I'm not using the library part `Common.Logging.Log4NetXXXX` anymore.
